tests/analysis/tensors/test_refresh.py

```diff
@@ -13,12 +13,10 @@
 @pytest.mark.slow  # 0:38 to run
 def test_refresh_tensors():
     trial_name = str(uuid.uuid4())
-    path = "s3://tornasole-testing/rules/tensors/ts_output/train/"
+    path = f"s3://smdebug-testing/outputs/rules_refresh_tensors-{uuid.uuid4()}/"
```

Comment:
all s3 tests should not be part of unit test run. is slow already doing this?

Reply:
Running `pytest tests --skipslow` will run a subset of the tests as a sanity check before passing CI. S3 tests are currently a part of CI.